src/checkpoint1/checkpoint1.css

```diff
@@ -0,0 +1,218 @@
+body {
+    display: grid;
+    height:100%;
+    width:100%;
+    grid-template-columns: repeat(20,1fr);
+    grid-template-rows:repeat(20,1fr);
+}
+
+header {
+    background-color: rgb(83, 77, 77);
+    color:rgb(241, 236, 236);
+    text-align: center;
+    grid-column: 1 / 20;
+    grid-row: 1 / 2;
+    text-align: right;
+    padding: 15px;
+    word-spacing: 70px;
+}
+
+aside {
+    background-color: rgb(37, 210, 223);
+    text-align: center;
+    font-size: 15px;
+    color:white;
+    grid-column: 1/3;
+    grid-row: 2/20;
+    display:flex;
+    flex-direction: column;
+    align-items: flex-start;
+    justify-content: space-around;
+    padding: 10px;
+}
+
+.title {
+    grid-column: 1/1;
+    grid-row: 1/1;
+    color:white;
+    font-size: 30px;
+    margin-left: 40px;
+    margin-top: 5px;
+}
+
+main {
+    background-color: lightgrey;
+    grid-column: 3/20;
+    grid-row: 2/20;
+}
+
+.dash {
+    font-size: 40px;
+    grid-column: 4/12;
+    grid-row: 2/2;
+    margin-top: 20px;
+    display: flex;
+
+}
+
+h6 {
+    font-size: 20px;
+    padding: 10px;
+    margin-top: 8px;
+    color: grey;
+
+}
+
+nav {
+    background-color: white;
+    grid-column: 4/19;
+    grid-row: 4/5;
+    word-spacing: 20px;
+    display: flex;
+}
+
+.icons {
+    background-color: white;
+    display: flex;
+    justify-content: space-around;
+}
+
+.one {
+    grid-column: 4/7;
+    grid-row: 6/10;
+    background-color: white;
+    display:flex;
+    flex-direction: column;
+    justify-content: space-around;
```

Comment:
this note is for all of these icon-price boxes and all future stuff. you never want the text to touch the edge of the container it's in. look at this github page you're on. look for elements with defined borders. you won't find an element with text touching it. best way to fix that is using `padding`! 
add `padding: 10px;` < == this adds padding all around. 
you could also do: `padding: 10px 10px 10px 10px;` <== same as above but you're defining the padding for each side. you start with top, right, bottom, left.  so you can adjust to your needs. 
or `padding-left: 10px;`.. etc. you get it.

Reply:
check out the html for more notes on this particular element